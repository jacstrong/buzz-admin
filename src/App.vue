<template>
  <v-app>
    <snackbar/>
    <router-view/>
  </v-app>
</template>

<script>
import snackbar from './components/Snackbar.vue'
import { mapMutations, mapGetters, mapActions } from 'vuex'

export default {
  components: {
    snackbar
  },
  methods: {
    ...mapMutations({
      setAuth: 'setAuth'
    }),
    ...mapActions({
      checkCurrentUser: 'checkCurrentUser'
    })
  },
  computed: {
    ...mapGetters({
      isLoggedIn: 'isLoggedIn'
    }),
  },
  beforeCreate() {
    if (!this.isLoggedIn) {
      // this.checkCurrentUser;
    }
  }
}
</script>

<style lang="scss">

</style>
