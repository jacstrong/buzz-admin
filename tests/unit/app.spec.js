// import Vue from 'vue'
import Vue from 'vue'
import { shallowMount } from '@vue/test-utils'
import Vuetify from 'vuetify'
import App from '@/App.vue'

Vue.use(Vuetify)

describe("App.test.js", () => {
    let cmp;

    beforeEach(() => {
        cmp = shallowMount(App, {

        });
    });

    it("has the expected html structure", () => {
        expect(cmp.element).toMatchSnapshot();
    });
});
