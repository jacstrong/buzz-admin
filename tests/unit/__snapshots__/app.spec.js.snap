// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.test.js has the expected html structure 1`] = `
<v-app>
  <snackbar-stub />
   
  <router-view />
</v-app>
`;
